features/task_validate_image.feature

```diff
@@ -257,6 +257,7 @@ Feature: Verify Enterprise Contract Tekton Tasks
     And the task logs for step "initialize-tuf" should match the snapshot
      And the task logs for step "report" should match the snapshot
      And the task logs for step "summary" should match the snapshot
+     And the task logs for step "show-config" should match the snapshot
```

Comment:
If you add something like this, there needs to be a corresponding snapshot file change somewhere in `features/__snapshots__/task_validate_image.snap` so it has something to match.

There's a way to generate it, but you could try creating it by hand. Look for the ` [Outputs are there:summary - 1]` and `[Outputs are there:assert - 1]` sections. Right between those you can add a `[Outputs are there:show-config - 1]` and inside it you can put the expected output, then the `---` delimiter.

You can start with any garbage in there, and then run the test. It will show you what it got and what was expected, which you can then use to update the snapshot file.

Reply:
@simonbaird Could you PTAL?

Reply:
Looks alright, but see my other comment about the content of the snapshot change.